dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/deprecated/tracker/trackedentity/AbstractTrackedEntityInstanceService.java

```diff
@@ -1275,7 +1275,8 @@ private void assignFileResource(
     if (trackedEntityAttribute.getValueType().isFile()) {
       FileResource fileResource = fileResourceService.getFileResource(attribute.getValue());
 
-      if (!fileResource.isAssigned() || fileResource.getFileResourceOwner() == null) {
+      if (fileResource != null
```

Comment:
Please add a failing test first, then fix the bug 😄

Reply:
fixed